<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElseSetting</class>
 <widget class="QDialog" name="ElseSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>527</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>173</y>
      <width>524</width>
      <height>180</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="yearEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="hourEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="yearLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="hourLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="monthEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="monthLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="minEdit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="minLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="dayEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="dayLabel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="secondEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>60</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="secondLabel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>100</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>400</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>140</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
